sympl/_components/plot.py

```diff
@@ -64,13 +63,11 @@ def store(self, state):
         """
         if self.interactive:
             self._fig.clear()
-            time.sleep(1e-5)
             fig = self._fig
         else:
             fig = plt.figure()
         self._plot_function(fig, copy_state(state))
-        plt.draw_all()
-        if self.interactive:
-            time.sleep(1e-5)  # necessary to draw, pause can be arbitrarily small
-        else:
+        plt.tight_layout()
```

Comment:
One more change, didn't notice this before. plt.tight_layout() should be given by the user in the plot function if they want it, so it shouldn't be called here. Also plt.tight_layout isn't guaranteed to apply tight layout to the figure used in this object, if other figures have been initialized after it. So I think we should remove this line.